OpenAPI/kiota/install.md

```diff
@@ -94,3 +97,34 @@ For details on running Kiota, see [Using the Kiota tool](using.md).
+
+[REST API Client Code Generator](https://github.com/christianhelle/apiclientcodegen) is a collection of Visual Studio C# custom tool code generators for OpenAPI specifications. This extension will install Kiota on-demand and will also add the required NuGet packages to build the generated code to the project. The generated code is created as a "code-behind" file to the OpenAPI specifications file in the .NET project. This extension offers same-day releases for new Kiota versions, but this requires updating the extension, which can be configured to be automatically.
+
+**Installation:**
+
+1. From Visual Studio (Windows), using the [Manage Extensions](https://learn.microsoft.com/visualstudio/ide/finding-and-using-visual-studio-extensions) dialog box (Tools -> Manage Extensions), search for extension  called [REST API Client Code Generator](https://marketplace.visualstudio.com/items?itemName=ChristianResmaHelle.ApiClientCodeGenerator2022). This extension is available for [Visual Studio 2022 for AMD64 and ARM64](https://marketplace.visualstudio.com/items?itemName=ChristianResmaHelle.ApiClientCodeGenerator2022) and [Visual Studio 2019](https://marketplace.visualstudio.com/items?itemName=ChristianResmaHelle.ApiClientCodeGenerator)
+
+   For Visual Studio for Mac, follow [these instructions](https://github.com/christianhelle/apiclientcodegen/blob/master/docs/VisualStudioForMac.md#installation)
+
+2. Click on the **Download** button. The extension will be installed after closing Visual Studio so you will need to do a restart
+
+**Usage:**
+
+To generate code using Kiota, Right click on a project or location under a project and select **Add -> New REST API Client -> Generate with Microsoft Kiota**
+
+![Add New REST API Client](./images/community/rest-api-client-generator/add-new.png)
+
+This will prompt you for a URL to download the OpenAPI specifications file.
+
+If the operation worked successfully then you should see a code-behind file for the OpenAPI specifications. Once you have generated code, all changes to the OpenAPI specifications file will trigger a re-generation of the code-behind file via the **KiotaCodeGenerator** custom tool
+
+![Custom Tool Experience](./images/community/rest-api-client-generator/custom-tool-experience.png)
+
+If you already have a OpenAPI specifications file in the project, then you can also generate code using Kiota by right clicking on the file then selecting **REST API Client Generator -> Generate with Microsoft Kiota**
+
+![Genrate code using Kiota](./images/community/rest-api-client-generator/generate-from-existing.png)
```

Comment:
@jasonjoh what do you think about adding the usage here? It'd be the only installation media we do it for.

Reply:
I see your point. We should definitely just link to the extension's usage documentation rather than list it out here.

Reply:
Thanks for confirming. @christianhelle do you mind making that update as well as the one for the docs link please? after this we should be good to merge.

Reply:
@baywet will do

Reply:
Done